data/herg_blockers/meta.yaml

```diff
@@ -0,0 +1,59 @@
+---
+name: herg_blockers
+description: "Human ether-à-go-go related gene (hERG) is crucial for the coordination\nof the heart's beating. Thus, if a drug blocks the hERG, it could\
+    \ lead to severe\nadverse effects. Therefore, reliable prediction of hERG liability in the early\nstages of drug design is quite important to reduce\
+    \ the risk of cardiotoxicity\nrelated attritions in the later development stages."
+targets:
+    - id: hERG_blocker
+      description: whether it blocks hERG (1) or not (0)
+      units:
+      type: boolean
+      names:
+          - noun: hERG blocker
+          - noun: hERG active compound
+          - noun: hERG active compound (<10uM)
+          - noun: human ether-à-go-go related gene (hERG) blocker
+          - verb: blocks hERG
+          - verb: blocks human ether-à-go-go related gene (hERG)
+          - verb: active against hERG (<10uM)
+          - verb: active against human ether-à-go-go related gene (hERG)
+      uris:
+          - http://purl.obolibrary.org/obo/MI_2136
```

Comment:
this is only referring to the binder - do we want to keep it?

Reply:
otherwise, lgtm!

Reply:
If you think this doesn't make sense, we can remove it.